src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  display: flex;
```

Comment:
Setting global styles on the body tag can lead to conflicts with other styles. Consider applying flex properties to a specific class or id instead.